doc/installation.md

```diff
@@ -13,8 +13,7 @@ To install the package for the current user, run this command:
 
 Where `<SOURCE>` is the location of the package source, e.g. a git repo:
 
-    pip3 install --user git+https://github.com/kernelci/kcidb.git
-
+    pip3 install --user git+https://git@github.com/kernelci/kcidb.git@main
```

Comment:
This doesn't really need changing, as it would pick the `main` branch by default. It's actually better to keep it this way in case the main branch name changes.